modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -239,6 +240,8 @@
   /** The default time before a piece of signed content expires. 2 Hours. */
   protected static final long DEFAULT_URL_SIGNING_EXPIRE_DURATION = 2 * 60 * 60;
 
+  private static final int CREATED_BY_UI_ORDER = 16;
```

Comment:
The set of metadata fields and their display order is fully configurable. Therefore, hard-coding the display order of a specific field seems quite smelly. If you really need an explicit display order for this field, it might worth consider to configure the field "createdBy" in the metadata catalog configuration (as readonly) and introducing a property type that indicates that the value won't be included in Dublincore catalog itself.